Formula/traefik.rb

```diff
@@ -0,0 +1,70 @@
+require "language/go"
+
+class Traefik < Formula
+  desc "A modern reverse proxy"
```

Comment:
There's an audit rule against starting the description with an indefinite article ("a" or "an")

Reply:
Done.
Same question as before, are those audit rules are published somewhere?